cognite/client/_api/vision.py

```diff
@@ -66,7 +66,7 @@ def extract(
         file_external_ids: list[str] | None = None,
         parameters: FeatureParameters | None = None,
     ) -> VisionExtractJob:
-        """Start an asynchronous job to extract features from image files.
+        """`Start an asynchronous job to extract features from image files. <https://developer.cognite.com/api#tag/Vision/operation/postVisionExtract>`_
```

Comment:
If you want these examples in the openapi spec, you'll need to run `scripts/generate_code_snippets.py` and commit the output to https://github.com/cognitedata/service-contracts/blob/master/versions/v1/python-sdk-examples.json